linera-chain/src/chain.rs

```diff
@@ -526,6 +538,9 @@ where
             block.height.try_add_one()?,
             now,
         )?;
+
+        // Log Prometheus metrics
+        NUM_BLOCKS_EXECUTED.with_label_values(&[]).inc();
```

Comment:
Then we can log the label values here

Reply:
oh interesting, so labels carry values?

Reply:
I'm looking at https://docs.rs/prometheus/latest/prometheus/macro.register_histogram_vec.html but it doesn't really help :)

Reply:
Maybe it's fine if we don't mix on-chain data (here block height) and counters.

Reply:
Yeah, documentation isn't great haha but basically `IntCounter` would work sort of like a regular ODS counter (with different aggregations available depending on the counter type), and `IntCounterVec` would maybe be closer to Scuba in the sense that you can define labels when you create the counter, which would basically be column names for values that you'll log together with the actual metric. And then you can filter by those label values, etc.
So let's say you log you `chain_id` when you create a counter, then when you log it you add a certain `chain_id` value to that column position. You could then for example filter the metrics for a specific chain, probably create dashboards with it, etc
It's pretty cool